src/components/CTAButton/CTAButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CTAButtonStyled } from './CTAButton.styles';
+
+const CTAButton = ({ cta, type }) => (
```

Comment:
I would probably rename the `cta` prop to children and export directly `CTAButtonStyled`. No need for this wrapper than